src/models/subscription.ts

```diff
@@ -213,23 +184,7 @@ export class Subscription extends Model {
 		return results[0] as SyncStatusCount[];
 	}
 
-	// TODO: need to remove "RepoJSON" as old code is now removed.  We can now just use RepoSyncState directly
-	async updateSyncState(updatedState: RepoSyncStateObject): Promise<Subscription> {
-		const state = merge(await RepoSyncState.toRepoJson(this), updatedState);
-		await RepoSyncState.updateFromRepoJson(this, state);
-		return this;
-	}
-
-	async updateRepoSyncStateItem(repositoryId: number, key: keyof RepositoryData | "repositoryCursor" | "repositoryStatus", value: unknown) {
-		// TODO: this is temporary until we redo sync
-		if (key === "repositoryStatus" || key === "repositoryCursor") {
-			await this.update({ [key]: value });
-		} else {
-			await RepoSyncState.updateRepoForSubscription(this, Number(repositoryId), key, value);
-		}
-		return this;
-	}
-
+	// TODO: remove this, not necessary
```

Comment:
Maybe explain why it's not necessary? Whoever looks at this TODO will ask exactly that.